docs/generators/java-helidon-client.md

```diff
@@ -39,7 +39,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |enumUnknownDefaultCase|If the server adds new enum cases, that are unknown by an old spec/client, the client will fail to parse the network response.With this option enabled, each enum will have a new case, 'unknown_default_open_api', so that when the server sends an enum case that is not known by the client/spec, they can safely fallback to this case.|<dl><dt>**false**</dt><dd>No changes to the enum's are made, this is the default option.</dd><dt>**true**</dt><dd>With this option enabled, each enum will have a new case, 'unknown_default_open_api', so that when the enum case sent by the server is not known by the client/spec, can safely be decoded to this case.</dd></dl>|false|
 |fullProject|If set to true, it will generate all files; if set to false, it will only generate API files. If unspecified, the behavior depends on whether a project exists or not: if it does not, same as true; if it does, same as false. Note that test files are never overwritten.| ||
 |groupId|groupId in generated pom.xml| |org.openapitools|
-|helidonVersion|Helidon version for generated code| |3.0.1|
+|helidonVersion|Helidon version for generated code| |4.0.6|
```

Comment:
What about the default be "4"?  Just an idea. Not sure what I think about that.

Reply:
I already changed (locally) one place so the default version is described as the highest released version that matches any supplied prefix. 

I am trying very hard to avoid needing to update the generators every time we release Helidon, and we won't need to do that if we can instead rely on the enhanced version handling in the PR.

This is another spot that I had not gotten to. Good to have it flagged.